jerry-core/lit/lit-magic-strings.ini

```diff
@@ -177,6 +177,7 @@ LIT_MAGIC_STRING_TO_FIXED_UL = "toFixed"
 LIT_MAGIC_STRING_UNSHIFT = "unshift"
 LIT_MAGIC_STRING_VALUE_OF_UL = "valueOf"
 LIT_MAGIC_STRING_DATAVIEW_UL = "DataView"
+LIT_MAGIC_STRING_ENDS_WITH = "endsWith"
```

Comment:
This should be places after `LIT_MAGIC_STRING_FUNCTION = "function"` to keep the alphabetic order.